### ISSUE TITLE
Update url path to website's homepage

### ISSUE THREAD

**Issue by Samy-33** [BitByte-TPC/TPC-website#72]
Right now the full url is `https://iiitdmj.ac.in/webix.iiitdmj.ac.in`. Looks ugly.

Can you guys, get it to something more like: `https://iiitdmj.ac.in/programming-club` or `https://iiitdmj.ac.in/bitbyte` or better  just `https://iiitdmj.ac.in/tpc`

**Comment by garg3133** [comment]
I had asked @Priyansh61 to get a new domain name (any of `tpc.iiitdmj.ac.in`, `theprogclub.iiitdmj.ac.in`, or `bitbyte.iiitdmj.ac.in` should work) from CC, with its `CNAME` record set to `https://iiitdmj.ac.in/webix.iiitdmj.ac.in`.

Would something like that work in our case?

**Comment by Samy-33** [comment]
I doubt that CNAME will help here. AFAIU, CNAME only allows mapping from a domain to another domain which doesn't include the `URL path`.

Having a sub-domain would require more effort than having a url path modified.

**Comment by garg3133** [comment]
You're right. CNAME record can only be used to point to another domain name (`iiitdmj.ac.in` for example) and not to the complete URL path (`https://iiitdmj.ac.in/webix.iiitdmj.ac.in` here).

So in that case, even if we get a sub-domain and set its CNAME record to point to `iiitdmj.ac.in`, it would be of no use since again we would need to use `https://bitbyte.iiitdmj.ac.in/webix.iiitdmj.ac.in/` to get to the actual TPC site, and moreover just hitting `bitbyte.iiitdmj.ac.in` would open the institute website.

So, we should just try to change the `/webix.iiitdmj.ac.in` part to something else for now (or keep that as it is and ask them to create a new path for the TPC website).

And then, maybe later, we can ask CC if they can setup a new sub-domain for us, for which I'm guessing they would need to create a new VirtualHost entry (assuming they're using Apache) on the server and set its `DocumentRoot` to point to the directory where TPC files are stored.
